src/main/java/seedu/address/model/person/Person.java

```diff
@@ -102,6 +103,18 @@ public Address getAddress() {
         return address.get();
     }
 
+    @Override
```

Comment:
May want to reorder the getters/setters to follow the ordering of current methods